themes/bootstrap3/less/components/search.less

```diff
@@ -838,3 +838,10 @@ h2.querySuggestions, h2.spellingSuggestions {
   line-height: @line-height-base;
   margin: 0;
 }
+
+h2.spellingSuggestions + ul {
+  padding-inline-start: 0;
+  > li {
+    list-style: none;
+  }
+}
```

Comment:
The styling is for backwards compatibility, although honestly I think why not make a list look like a list?

Reply:
I also wonder if it would be better to just put a class on the outer list tag instead of relying on the positioning relative to the h2; that seems a bit fragile.

Reply:
Did that!

Reply:
And I decided I think the top level one should actually look like a list, but keep the second level inline.